skr/webhook/chart/skr-webhook/templates/secret.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: skr-webhook-tls
+type: Opaque
+data:
+  {{- $data := (lookup "v1" "Secret" .Release.Namespace (printf "%s-webhook-tls" .Release.Name )).data }}
```

Comment:
this lookup requires the secret to be present already as I understand when you install the template. Thats why we have to be careful. If it can determine all of this dynamically, this comment does not exist :D

Reply:
it's dynamic in nature :)

Reply:
I just checked and this only works if you already have the secret present before you render the chart. it will fail otherwise and output empty data